LICENSES/Apache-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
do you have to keep the blank empty line?

Reply:
yeah I'd rather keep the file _exactly_ the same as the canonical/steward's license, i.e https://www.apache.org/licenses/LICENSE-2.0.txt

Reply:
makes sense